tests/integration/StatisticsLayoutTest.php

```diff
@@ -0,0 +1,235 @@
+<?php
+/**
+ * Copyright (C) 2019 Xibo Signage Ltd
+ *
+ * Xibo - Digital Signage - http://www.xibo.org.uk
+ *
+ * This file is part of Xibo.
+ *
+ * Xibo is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * any later version.
+ *
+ * Xibo is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with Xibo.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+namespace Xibo\Tests\Integration;
+
+use Xibo\OAuth2\Client\Entity\XiboDisplay;
+use Xibo\OAuth2\Client\Entity\XiboLayout;
+use Xibo\OAuth2\Client\Entity\XiboLibrary;
+use Xibo\OAuth2\Client\Entity\XiboStats;
+use Xibo\Tests\Helper\DisplayHelperTrait;
+use Xibo\Tests\Helper\LayoutHelperTrait;
+use Xibo\Tests\LocalWebTestCase;
+
+/**
+ * Class StatisticsLayoutTest
+ * @package Xibo\Tests\Integration
+ */
+class StatisticsLayoutTest extends LocalWebTestCase
+{
+
+    use LayoutHelperTrait, DisplayHelperTrait;
+
+    protected $startMedias;
+    protected $startLayouts;
+    protected $startDisplays;
+
+    /**
+     * setUp - called before every test automatically
+     */
+    public function setup()
+    {
+        parent::setup();
+        $this->startMedias = (new XiboLibrary($this->getEntityProvider()))->get(['start' => 0, 'length' => 10000]);
```

Comment:
this records the "state of the system" - query if we need that for this test - see below comments.